.devcontainer/devcontainer.json

```diff
@@ -1,40 +1,40 @@
 {
-  "build": {
-    "dockerfile": "./Dockerfile",
-    "context": "."
-  },
+  "dockerComposeFile": "./docker-compose.yml",
+  "service": "auto-gpt",
+  "workspaceFolder": "/workspace/Auto-GPT",
+  "shutdownAction": "stopCompose",
   "features": {
     "ghcr.io/devcontainers/features/common-utils:2": {
       "installZsh": "true",
-      "username": "vscode",
-      "userUid": "1000",
-      "userGid": "1000",
-      "upgradePackages": "true"
+        "username": "vscode",
+        "userUid": "6942",
+        "userGid": "6942",
+        "upgradePackages": "true"
     },
     "ghcr.io/devcontainers/features/desktop-lite:1": {},
+    "ghcr.io/devcontainers/features/node": "none",
```

Comment:
why this change?

Reply:
Was disabling lines one by one to find out why it was crashing, put that one back incorrectyly. Switched.